bucket/sftpgo.json

```diff
@@ -0,0 +1,36 @@
+{
+    "version": "2.1.0",
+    "description": "Fully featured and highly configurable SFTP server with optional FTP/S and WebDAV support - S3, Google Cloud Storage, Azure Blob.",
+    "homepage": "https://github.com/drakkan/sftpgo",
+    "license": "AGPL-3.0",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/drakkan/sftpgo/releases/download/v2.1.0/sftpgo_v2.1.0_windows_portable_x86_64.zip",
+            "hash": "91cc2c263c59a8ba9d82dd12d221d1f44c3e43c7b6ffe0657bda12fcdabf70e6"
+        }
+    },
+    "bin": "sftpgo.exe",
+    "env_set": {
+        "SFTPGO_CONFIG_DIR": "$dir\\config",
+        "SFTPGO_HTTPD__TEMPLATES_PATH": "$dir\\templates",
+        "SFTPGO_HTTPD__STATIC_FILES_PATH": "$dir\\static",
+        "SFTPGO_HTTPD__BACKUPS_PATH": "$dir\\backups"
+    },
+    "persist": [
+        "config",
+        "backups"
+    ],
+    "post_install": [
+        "if (! (Test-Path \"$dir\\config\\sftpgo.json\")) { Copy-Item -Path \"$dir\\sftpgo.json\" -Destination \"$dir\\config\\sftpgo.json\" }",
+        "if (! (Test-Path \"$dir\\config\\sftpgo.db\")) { Copy-Item -Path \"$dir\\sftpgo.json\" -Destination \"$dir\\config\\sftpgo.db\" }",
+        "Remove-Item \"$dir\\sftpgo.json\", \"$dir\\sftpgo.json\" -Force"
```

Comment:
Should be `"Remove-Item \"$dir\\sftpgo.db\", \"$dir\\sftpgo.json\" -Force"`

Reply:
thx. I could only work on macOS during working hours😂